server/models/visit.js

```diff
@@ -0,0 +1,157 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var q = require('q');
```

Comment:
oh, I remember what it was. The friggin linter complains about `Q(...)`, thinking it's a constructor. I'll just leave it for the moment, probably downcase the other `Q`s at some point